src/Application/Config.php

```diff
@@ -23,4 +25,26 @@ public function combine( Config $config ): self {
 		);
 	}
 
+	public function getInstanceOfId(): PropertyId {
+		if ( $this->instanceOfId === null ) {
+			throw new RuntimeException( 'No instance of ID configured' );
+		}
+
+		return $this->instanceOfId;
+	}
+
+	/**
+	 * TODO: make private?
+	 */
+	public function getFacets(): FacetConfigList {
+		return $this->facets ?? new FacetConfigList();
```

Comment:
I ended up adding the below `getFacetConfigForInstanceType`, which is what made me put the TODO on this function.

Reply:
We might need both, or at least another version. See https://github.com/ProfessionalWiki/WikibaseFacetedSearch/pull/41/files#r1885911926